source/strings.tex

```diff
@@ -4836,6 +4872,22 @@
 Otherwise, returns \tcode{npos}.
 \end{itemdescr}
 
+\rSec2[string.view.deduction]{Deduction guide}
```

Comment:
This should be [string.view.deduct] to keep it short. I've already added an item to our style guide.